index.html

```diff
@@ -11,7 +11,7 @@
   </head>
 
   <body>
-
+    <!-- reviewing the code for linter errors and best practices -->
     <!-- <p>
         Hello World, We are Software devolopers!
     </p> -->
```

Comment:
please don't leave your unused code as a comment in the codebase, it makes your code not clean.

Reply:
- [ ] This is a good way to  phrased the comments politely, as suggestions instead of orders.